Tests/DependencyInjection/Factory/AbstractStorageFactoryTest.php

```diff
@@ -93,7 +93,7 @@ protected function assertDefinitionContainsMethodCall(Definition $serviceDefinit
     /**
      * @return AbstractStorageFactory|MockObject
      */
-    protected function createAbstractStorageFactory()
+    protected function createAbstractStorageFactory(): AbstractStorageFactory|MockObject
```

Comment:
I think technically this should be `AbstractStorageFactory&MockObject` based on the annotations in PHPUnit

https://github.com/sebastianbergmann/phpunit/blob/473f8020e428c5a2ddf6db60ae162c2b995018b5/src/Framework/MockObject/MockBuilder.php#L92

Although intersection types are only available in PHP 8.1

Reply:
You are right. I changed it. PhpStorm tooks it from the Annotation. But that was probably from me too.